src/core/startup.js

```diff
@@ -7,7 +7,7 @@
  * @author BBC Children's D+E
  * @license Apache-2.0 Apache-2.0
  */
-import { settings, settingsChannel } from "../core/settings.js";
+import { settings, settingsChannel, setGame } from "../core/settings.js";
```

Comment:
Could do with a better name here as setGame doesn't tell much about what it applies to on it's own. Maybe _settingsInit_? What do you think?

would have been nicer if we could have hung it all of settings, e.g: settings.init but then you'd have settings.settings and have to rename a lot of stuff :)